File: dotnet/CudaRS.Native/SdkYoloPreprocessMeta.cs
using System.Runtime.InteropServices;

namespace CudaRS.Native;

[StructLayout(LayoutKind.Sequential)]
public struct SdkYoloPreprocessMeta
{
    public float Scale;
    public int PadX;
    public int PadY;
    public int OriginalWidth;
    public int OriginalHeight;
}

File: cudars-ffi/src/sdk/sdk_yolo_preprocess_meta.rs
#[repr(C)]
#[derive(Debug, Clone, Copy, Default)]
pub struct SdkYoloPreprocessMeta {
    pub scale: f32,
    pub pad_x: i32,
    pub pad_y: i32,
    pub original_width: i32,
    pub original_height: i32,
}
